active_record_doctor.gemspec

```diff
@@ -16,9 +16,9 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE.txt", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", "~> 4.2"
+  s.add_dependency "rails", "< 5.1", ">= 4.2"
 
-  s.add_development_dependency "rails", "~> 4.2"
+  s.add_development_dependency "rails", "< 5.0", ">= 4.2"
```

Comment:
Surprisingly it installed without any problems with Rails 5.x, however

`< 5.1` sure is more future proof.